components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java

```diff
@@ -0,0 +1,269 @@
+     *
+     * @param dbType database type string.
+     * @return true if the database type matches the driver type, false otherwise.
+     * @throws OrganizationManagementServerException If error occurred while checking the DB metadata.
+     */
+    private static boolean isDBTypeOf(String dbType) throws OrganizationManagementServerException {
+
+        try {
+            NamedJdbcTemplate jdbcTemplate = getNewTemplate();
+            return jdbcTemplate.getDriverName().toLowerCase().contains(dbType) ||
+                    jdbcTemplate.getDatabaseProductName().toLowerCase().contains(dbType);
+        } catch (DataAccessException e) {
+            throw handleServerException(ERROR_CODE_ERROR_CHECKING_DB_METADATA, e);
+        }
+    }
+
+    /**
+     * Get the tenant ID.
+     *
+     * @return the tenant ID.
+     */
+    public static int getTenantId() {
+
+        return PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId();
+    }
+
+    /**
+     * Get the tenant domain.
+     *
+     * @return the tenant domain.
+     */
+    public static String getTenantDomain() {
+
+        return PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+    }
+
+    /**
+     * Get the organization id from context.
+     *
+     * @return the organization id.
+     */
+    public static String getOrganizationId() {
+
+        return PrivilegedCarbonContext.getThreadLocalCarbonContext().getOrganizationId();
+    }
+
+    /**
+     * Get the username of the authenticated user.
+     *
+     * @return the username of the authenticated user.
+     */
+    public static String getAuthenticatedUsername() {
+
+        return PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername();
+    }
+
+    /**
+     * Get the user ID.
+     *
+     * @return the user ID.
+     */
+    public static String getUserId() {
+
+        return PrivilegedCarbonContext.getThreadLocalCarbonContext().getUserId();
+    }
+
+    /**
+     * Build URI prepending the server API context with the proxy context path to the endpoint.
+     *
+     * @param organizationId The organization ID.
+     * @return Relative URI.
+     */
+    public static String buildURIForBody(String organizationId) {
+
+        String context = getContext(V1_API_PATH_COMPONENT + PATH_SEPARATOR + ORGANIZATION_PATH
+                + PATH_SEPARATOR + organizationId);
+
+        return context;
+    }
+
+    /**
+     * Builds the API context.
+     *
+     * @param endpoint Relative endpoint path.
+     * @return Context of the API.
+     */
+    public static String getContext(String endpoint) {
+
+        String organizationId = getOrganizationId();
+        if (StringUtils.isNotBlank(organizationId)) {
+            return String.format(ORGANIZATION_CONTEXT_PATH_COMPONENT, organizationId) + SERVER_API_PATH_COMPONENT +
+                    endpoint;
+        } else {
+            return SERVER_API_PATH_COMPONENT + endpoint;
+        }
```

Comment:
else is redundant

Reply:
Else block handles the case if organization ID is not set in the `PrivilegedCarbonContext`

Reply:
what I meant was we can straight away `return SERVER_API_PATH_COMPONENT + endpoint;` instead of adding an else

Reply:
resolved by 94ae21c9175e4ed76f7da45ced9bf1fb626247c4